README.md

```diff
@@ -1,7 +1,8 @@
 [![open-quantum-safe](https://circleci.com/gh/open-quantum-safe/oqs-demos.svg?style=svg)](https://app.circleci.com/pipelines/github/open-quantum-safe/oqs-demos)
 
-oqs-demos
-=========
+# oqs-demos
```

Comment:
For level 1 and 2 headings, I like using `====` and `----` in Markdown files so that people who read them as plain text files get nice underlines (rather than the semantically equivalent but visually less distinguished `#` and `##` syntax).  (For level 3 headings, I believe `###` is the only supported syntax in Markdown so I use that.)  But this is not worth revising the PR over.

Reply:
Thanks for letting me know about this convention. Will try to heed it -- for 1st level headings: 2nd levels are hardly anywhere marked with ---. Might make sense to add to https://github.com/open-quantum-safe/liboqs/wiki/Contributing-guide if you'd like to see this generally applied.

Reverted here to maintain convention.